### ISSUE TITLE
Error messages in GG 1.2.2 when opening an existing file created with GG 1.2.1

### ISSUE THREAD

**Issue by charliehoward4dp** [DistributedProofreaders/guiguts#459]
The message.log popped up with information in the attached file the first time I opened an existing html file (from GG 1.2.1) using File->Open with just-installed, uncustomized GG 1.2.2. GG did not crash, and it showed the file properly. That file and its bin also are in the attached zip.
[bugreportv122.zip](https://github.com/DistributedProofreaders/guiguts/files/5792945/bugreportv122.zip)

I wrote the above part of this report, closed GG without changing anything, restarted it, and chose the same html file, but this time from the Recent Files List.  There were no error messages.

Closed GG, restarted it, and chose the same html file using File->Open. There were no error messages (unlike the first time).

Closed it yet again, put a copy of 'setting.rc' into the attached zip, deleted that file, restarted GG 1.2.2, chose the same html file using File->Open.  There were no error messages.

**Comment by windymilla** [comment]
Did you edit the setting.rc file manually to ensure you got the new PPWB link?
I ask because the code that gave the error is trying to create the Custom Menu, and obviously found something wasn't initialised correctly.
I'm pretty sure it's nothing to do with File->Open.
That would also explain why it only happened once.

**Comment by charliehoward4dp** [comment]
Did not edit setting.rc, did not copy it from an earlier version of GG. (This just confirms my private message, which I don't see here, and am reproducing below):

No, I didn't edit setting.rc, nor copy my old one.  I opened the downloaded 1.2.2 zip, dragged the "Guiguts" folder to the folder where I keep all versions, renamed the new "Guiguts" folder to "GG 1.2.2", went into it, and clicked the "run" .bat file.

Next, I brought up the Preferences>Processing menu and took a Screen Shot (for documentation). I didn't change any settings.

Next, I brought up the Custom Menu (successfully) and took a Screen Shot.

Next, I brought up the Custom Menu again and clicked "Set External Programs" to take a Screen Shot of that. Didn't change anything, but of course had to click "OK" to get out of it. The "Message Log" did not pop up at this point.

Next, I went to the HTML Menu, clicked "HTML Generator" and "Auto Illus Search" (to get Screen shots of the new Image Dialog).  Nothing happened, must be because I didn't have an open file.

Next, I think I may have closed and restarted GG, but now am not certain that I closed/restarted it before going to the File menu to get an HTML file for "Auto Illus Search."

Next, I clicked GG's "File" menu, selected "Open...", navigated to my most recently-finished project, and double-clicked an .html file that was ready to have illustration tags added to it. That's when the Message Log popped up.
